lib/fog/openstack/docs/metering.md

```diff
@@ -20,9 +20,23 @@ service = Fog::Metering::OpenStack.new({
 
 ## Events
 
-* `service.events`: Return a list of events.
+* `service.events([<query_filter>])`: Return a list of events.
 * `service.events.find_by_id(<message_id>)`: Return the event matching message_id, or nil if no such event exists.
 
+### Filter events example
+
+Return events newer than 2016-03-17T09:59:44.606000.
+
+```ruby
+query_filter = [{
+  'field' => 'start_timestamp',
+  'op'    => 'gt',
+  'value' => '2016-03-17T09:59:44.606000'
```

Comment:
You can achieve this time format using

```ruby
Time.now.utc.iso8601(6)
```

Helpful for people who are passing Ruby time objects around their codebase.